From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: build-linux CI <ci@localhost>
Date: Sat, 1 Jan 2000 00:00:00 +0000
Subject: [PATCH 1/2] Add localversion marker

Append a marker to the kernel release string so that tests can verify
that patches were applied to the kernel that ended up being built.
---
 localversion-test | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 localversion-test

diff --git a/localversion-test b/localversion-test
new file mode 100644
index 000000..a4661c
--- /dev/null
+++ b/localversion-test
@@ -0,0 +1 @@
+-patched
